docs/run-operator-locally.md

```diff
@@ -4,26 +4,43 @@ This document contains a quickstart guide to build and running (+debugging) the
 Being able to run and build the binary locally can help with faster feedback-cycles.
 
 ## Prerequisites
-- follow the general setup to be able to run e2e tests locally with our suite as described here, which includes the usage of [telepresence](https://www.getambassador.io/docs/telepresence/latest/quick-start/):
+- Follow the general setup to be able to run e2e tests locally with our suite as described here, which includes the usage of [telepresence](https://www.getambassador.io/docs/telepresence/latest/quick-start/):
   - [contributing.md](contributing.md)
   - [build_operator_locally.md](build_operator_locally.md)
-  - if above has been configured there should be either:
+  - If above has been configured there should be either:
     - `$HOME/.kube/config`
     - `KUBECONFIG` environment variable pointing at a file
     - **Note**: either of these are necessary to be able to run the operator locally
-- have a folder `.community-operator-dev`
-- this guides uses intellij as an example. Running any other way should be very similar.
+- Have a folder `.community-operator-dev`
 ## Goals
-- runs the operator locally as a binary (optionally in debug mode) in with telepresence
-- runs e2e tests locally 
-  - **Note:** 
-    - if you plan to run the e2e tests, there are sub-steps that will install the following helm-chart: [operator.yaml](helm-charts/charts/community-operator/templates/operator.yaml)
-    - by default, the template chart contains and the operator with `1` replica. This will bite with our local running operator. With this in mind the solution is to set the number to `0`
-
-## Steps for Intellij
-- run the make target creating env file `make generate-env-file` 
-  - which creates/updates an env file `local-test.env` in .community-operator-dev/
-- use said env file in intellij while running the main of the operator: `cmd/manager/main.go`
+- Run the operator locally as a binary (optionally in debug mode) in command line or in an IDE
+- Run e2e tests locally
+
+## Running The Operator locally - Intellij
+1. Run the make target creating env file `make generate-env-file`, which creates/updates an env file `local-test.env` in .community-operator-dev/
+2. Use local-test.env file in IntelliJ while running the main of the operator: `cmd/manager/main.go`
```

Comment:
I'm still missing here a reference to EnvFiles plugin. Without this there is no way to use env files.

Reply:
Ah you are totally right. I overlooked this. On it

Reply:
Somehow I lost track of it, in any case: 1f7830b